Dapper NET40/SqlMapper.cs

```diff
@@ -5195,18 +5195,14 @@ public void AddParameter(IDbCommand command, string name)
     /// </summary>
     partial class FeatureSupport
     {
-        private static readonly FeatureSupport
-            @default = new FeatureSupport(false),
-            postgres = new FeatureSupport(true);
-
         /// <summary>
-        /// Gets the feature set based on the passed connection
+        /// Gets the featureset based on the passed connection
         /// </summary>
         public static FeatureSupport Get(IDbConnection connection)
         {
             string name = connection == null ? null : connection.GetType().Name;
-            if (string.Equals(name, "npgsqlconnection", StringComparison.OrdinalIgnoreCase)) return postgres;
-            return @default;
+            if (Config.GetDbmsType(connection) == DbmsType.Postgres) return new FeatureSupport(true); //Postgresql supports arrays
+            return new FeatureSupport(false);

```

Comment:
This change causes an extra allocation for every call, the previous static cache was explicitly to avoid allocations.